examples/htool_spi_proxy.h

```diff
@@ -25,7 +25,7 @@ struct htool_spi_proxy {
   bool is_4_byte;
 };
 
-int htool_spi_proxy_init(struct htool_spi_proxy* spi, struct libhoth_device* dev);
+int htool_spi_proxy_init(struct htool_spi_proxy* spi, struct libhoth_device* dev, bool noenter4b);
```

Comment:
Variable name in definition does not match declaration. Is that intended for documentation reasons?

Reply:
Actually, they are contradictory. `noenter4b` != `is_4_byte`